build_library/vm_image_util.sh

```diff
@@ -590,6 +595,20 @@ _write_qemu_uefi_conf() {
     VM_GENERATED_FILES+=( "$(_dst_dir)/${ovmf_ro}" "$(_dst_dir)/${ovmf_rw}" )
 }
 
+_write_qemu_uefi_secure_conf() {
+    local tmpdir=`mktemp -d`
```

Comment:
You can just use `${VM_TMP_DIR}` instead of creating a new dir, that way failures don't leave things scattered around.